### PR TITLE
[14.0] [MIG] maintenance_plan_activity

### PR DESCRIPTION
**Pull request by mariadforgeflow**
Standard migration to v14

### CODE REVIEW

File: maintenance_plan_activity/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+{
+    "name": "Maintenance Plan Activity",
+    "summary": """
+        This module allows defining in the maintenance plan activities that
+        will be created once the maintenance requests are created as a
+        consequence of the plan itself.""",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "maintainers": ["AdriaGForgeFlow "],
```
Comment (etobella) [inline maintenance_plan_activity/__manifest__.py]: I thinkbthere is an extra space

**Review by etobella — COMMENTED** [review]
Just a minor comment

File: maintenance_plan_activity/__manifest__.py (hunk 1 above)
Comment (mariadforgeflow) [reply maintenance_plan_activity/__manifest__.py]: True, my bad! Thank you

**Review by ChristianSantamaria — APPROVED** [review]
Locally tested, LGTM 👍

**Review by etobella — APPROVED** [review]
Direct migration

**Review by DavidBForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**